paper.tex

```diff
@@ -226,7 +226,21 @@
 
 % (http://www.easterbrook.ca/steve/2010/01/how-to-write-a-scientific-abstract-in-six-easy-steps/)
 
-\lipsum[1]
+  % generated by lipsum
```

Comment:
I had to create some nuance in the abstract (paragraphs and math) and also use the expanded text from `lipsum` otherwise it gets removed by `pandoc`.

Reply:
Have you tried it with our xDSL paper?
Is `\toolname` being replaced correctly, or is also removed by `pandoc`?

Reply:
Great point, I moved everything to Python and `\toolname{}` and along other similar definitions work now.
Would it make sense to have a `requirements.txt`? Not sure why we don't have one for the `tools` so far.

Reply:
@math-fehr ping